Plugin/src/SofaPython3/PythonEnvironment.h

```diff
@@ -30,7 +30,18 @@
 #pragma push_macro("slots")
 #undef slots
 /// This should come from python3.
+#if defined(_DEBUG) && defined(WIN32)
+#define DONT_FORGET_TO_ENABLE_DEBUG
+#endif // defined(_DEBUG) && defined(WIN32)
+
+#undef _DEBUG //Prevent linking debug build of python
 #include <Python.h>
+
+#ifdef DONT_FORGET_TO_ENABLE_DEBUG
+#define _DEBUG 1
+#undef  DONT_FORGET_TO_ENABLE_DEBUG
+#endif
+
```

Comment:
Not mandatory but looks better to me 

```suggestion
#if defined(WIN32) && defined(_DEBUG)
    #undef _DEBUG // Prevent linking debug build of python
    #include <Python.h>
    #define _DEBUG 1
#else
    #include <Python.h>
#endif

```

Reply:
seems good to me (even if i like my superbly named macro)